### PR TITLE
Implemented contact section

### LINKED ISSUE #7: Implement Contact section
# Context

simian-contact

### PR DESCRIPTION
**Pull request by TheMightyPenguin**
# Context

This section was implemented by:

- Made a notify form section (for future reuse, as this is also used in the lead)
- Added contact component with styling

This closes #7 

## Media

contact

### CODE REVIEW

File: _data/notify-form.yml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+email_placeholder: Email Address
+button_Text: Notify me
```
Comment (duranmla) [inline _data/notify-form.yml]: `button_text` keep the naming as you are doing already

File: _includes/notify_form.html [hunk 2]
```diff
@@ -0,0 +1,18 @@
+{% assign form = site.data.notify-form %}
+
+<form class="notify-form form-inline">
+  <div class="row text-center">
+
+    <div class="col-12 col-lg">
+      <label class="sr-only" for="email">Email</label>
```
Comment (duranmla) [inline _includes/notify_form.html]: this is not visible right? as I didn't see it within the gif.

File: _sass/_objects--buttons.scss [hunk 3]
```diff
@@ -0,0 +1,17 @@
+$size--horizontal-padding:                $spacer*2.5;
+
+.btn {
+  font-family: $primary-font;
```
Comment (duranmla) [inline _sass/_objects--buttons.scss]: Are you sure u can't set this within bootstrap variables?

File: _sass/_objects--buttons.scss [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+}
+
+.btn-primary {
+  font-size: 22px;
+  font-weight: 600;
+  padding-left: $size--horizontal-padding;
+  padding-right: $size--horizontal-padding;
+
+  &:hover {
+    background-color: $jade;
```
Comment (duranmla) [inline _sass/_objects--buttons.scss]: Bootstrap variables for hover color, besides, check bootstrap variables for font-size and font-weight (event padding horizontal has its variable like padding-x or something like that) on top of that use font-size as `rem` and using `font-size-base` from bootstrap or a variable that match your need

File: _sass/_objects--notify-form.scss [hunk 5]
```diff
@@ -0,0 +1,18 @@
+$size--input-height:                          45px;
+$size--input-width:                           305px;
+$color--input-border:                         $alto;
+
+.notify-form {
```
Comment (duranmla) [inline _sass/_objects--notify-form.scss]: I like it `contact` and within a `notify-form` 👍 , HOWEVER, maybe the `contact` will be more like a `layout` or even `partial`? I mean you define positions and special styles for that section and whenever the section is displayed will shape its children like that

File: _sass/_objects--notify-form.scss [hunk 6]
```diff
@@ -0,0 +1,18 @@
...
+  &-input {
+    display: inline-block;
+    height: $size--input-height;
```
Comment (duranmla) [inline _sass/_objects--notify-form.scss]: almost everything here can be set with bootstrap variables

File: _sass/_objects--notify-form.scss [hunk 7]
```diff
@@ -0,0 +1,18 @@
...
+    padding: 0 $input-padding-x;
```
Comment (duranmla) [inline _sass/_objects--notify-form.scss]: So you saw the variable, maybe the doubt is if you should set the whole input style for all the app, but I will say yes, your input forms will look like the one within the contact section

File: _sass/_settings--bootstrap.scss [hunk 8]
```diff
@@ -4,9 +4,16 @@
  * to be used by Bootstrap to delivery its grid and
  * overall styles. A complete list of variables
  * can be found at: (http://getbootstrap.com/customize/#less-variables)
```
Comment (duranmla) [inline _sass/_settings--bootstrap.scss]: Actually the link here is wrong, yours is better the comment that I set for a recent project was:

```
Add into this file the variables that are going
  to be used by Bootstrap to delivery its grid and
  overall styles. A complete list of variables
  can be found within the bootstrap module at node_modules
  within the _variables.scss file
```

I think we should use it and append the link you mentioned

File: _sass/_settings--bootstrap.scss [hunk 9]
```diff
@@ -4,9 +4,16 @@
...
+ * and also: (https://v4-alpha.getbootstrap.com/getting-started/options/)
  */
 
 /**
+ *
```
Comment (duranmla) [inline _sass/_settings--bootstrap.scss]: What section is this in bootstrap variables?

File: _sass/_settings--palette.scss [hunk 10]
```diff
@@ -2,12 +2,18 @@
  * Color Variables
  */
 
-$tundora:                             #4a4a4a;
```
Comment (duranmla) [inline _sass/_settings--palette.scss:5]: let's keep each sections of colors alphabetically sorted

**Review by duranmla — CHANGES_REQUESTED** [review]
We are improving a lot, I had my doubts regarding some decisions you made that I think you can keep using the bootstrap variables but it is cool I see you are actually consulting the variables soI believe that is a matter of decision and not cause you forget to check the vars.

File: _includes/notify_form.html [hunk 11]
```diff
@@ -0,0 +1,18 @@
+{% assign form = site.data.notify-form %}
+
+<form class="notify-form form-inline">
```
Comment (duranmla) [inline _includes/notify_form.html]: BTW whenever the button of submit stays below the input the button and the input should be width 100% as two blocks

File: _includes/notify_form.html (hunk 2 above)
Comment (TheMightyPenguin) [reply _includes/notify_form.html]: Nope! Only for screenreaders, should I remove the label?

File: _sass/_objects--buttons.scss [hunk 12]
```diff
@@ -0,0 +1,13 @@
...
+  // FIXME: This should be set automaticaly by bootstrap using some variable/setting
+  font-family: $primary-font;
+  cursor: pointer;
+}
+
+.btn-primary {
+  font-size: $font-size-base*1.375;
+
+  &:hover, &:focus {
+    background-color: $jade;
```
Comment (duranmla) [inline _sass/_objects--buttons.scss]: @TheMightyPenguin this seems to be something worth having in a variable

**Review by duranmla — APPROVED** [review]
:100:

**Pull request merged**